pkg/db/db.go

```diff
@@ -182,8 +182,8 @@ func (c *Client) updateDownloadedAt(dst string) error {
 	return nil
 }
 
-func (c *Client) populateOCIArtifact() error {
-	if c.artifact == nil {
+func (c *Client) populateOCIArtifact(force bool) error {
+	if c.artifact == nil || force {
```

Comment:
I found `force` is always true... Can we simply populate it anytime by removing the if-statement?

Reply:
> I found `force` is always true... Can we simply populate it anytime by removing the if-statement?

yes, I wanted to write the same comment.

Reply:
We should test it carefully, image scanning, client/server mode, Kubernetes scanning, etc. I don't remember, but there was a reason to avoid multiple initializations. It may be no longer necessary after some refactoring though.